package.json

```diff
@@ -6,7 +6,8 @@
     "dev": "partykit dev",
     "deploy": "partykit deploy",
     "build": "tsc --noEmit",
-    "build:extension": "node esbuild.mjs"
+    "build:extension": "node esbuild.mjs",
+    "build:extension:prod": "PARTY_HOST=https://github-party.partykit.dev node esbuild.mjs"
```

Comment:
Is this the real url? It doesn't look like this resolves to a real dns address

<img width="915" height="363" alt="Image" src="https://github.com/user-attachments/assets/6f1f8c77-fc03-4ad0-9033-db52dd23d130" />